x/tokenomics/types/errors.go

```diff
@@ -3,10 +3,18 @@ package types
 // DONTCOVER
 
 import (
-	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+	sdkerrors "cosmossdk.io/errors"
 )
 
 // x/tokenomics module sentinel errors
 var (
-	ErrSample = sdkerrors.Register(ModuleName, 1100, "sample error")
+	ErrTokenomicsAuthorityInvalidAddress    = sdkerrors.Register(ModuleName, 1, "provided authority address is invalid")
+	ErrTokenomicsAuthorityAddressIncorrect  = sdkerrors.Register(ModuleName, 2, "provided authority address is incorrect")
```

Comment:
`ErrTokenomicsAuthorityInvalidAddress` and `ErrTokenomicsAuthorityAddressIncorrect` seem semantically too close to me, maybe rephrasing them or providing more context in the error message would help getting it without the need to look at their usage.